Module4_Labs/Lab7_Zoologist/Student-Starter/test_cases_check1.py.py

```diff
@@ -0,0 +1,21 @@
+"""
+Test Case 1
+input: Animals.txt
+output: {0: 'Horse', 1: 'Buffalo', 2: 'Cat', 3: 'Rabbit', 4: 'Goat', 5: 'Dog', 6: 'Sheep', 7: 'Pig', 8: 'Cow', 9: 'Human', 10: 'Rat', 11: 'Mice', 12: 'Bat', 13: 'Squirrel', 14: 'Turtle'}
+"""
+
+"""
+Test Case 2
+input: print[equations]
+output: [1, 32, 8.0, 3, 0.0, 1.414213562373095, 0.1, 0.0, 6, 0.0, 1, 100.0, 1.0, 1, 1.414213562373095]
+"""
+
+"""
+Test Case 3
+input: on the start of the program
+output:
+Please chose a sorting method:
+1.(Bubble) Sort
+2.(Insertion) Sort
+3.(Selection) Sort
```

Comment:
can you add 1 or 2 more test cases, testing that the menu has the correct validation in it? like "boobly" or "inertia" should fail and prompt the program to reprint the menu